### PR TITLE
select layers with chips

### PR DESCRIPTION
**Pull request by fsdavid**

### CODE REVIEW

File: src/atlasComponents/sapiViews/core/atlas/chip/atlas.chip.component.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import {Component, EventEmitter, Input, Output} from "@angular/core";
```
Comment (xgui3783) [inline src/atlasComponents/sapiViews/core/atlas/chip/atlas.chip.component.ts:1]: when you do a lot of copy pasting of code, it is a good sign that this component can be extracted into a general purpose component (in this case, for atlas, space and parcellation)

File: src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import {Component, EventEmitter, Input, Output} from "@angular/core";
```
Comment (xgui3783) [inline src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts:1]: ditto above re: extracting into generalised component

File: src/viewerModule/viewerCmp/viewerCmp.component.ts [hunk 3]
```diff
@@ -104,6 +106,13 @@ export class ViewerCmp implements OnDestroy {
     map(({ parcellation }) => parcellation)
   )
 
+  public allAvailableAtlas$ = this.store$.pipe(
+    atlasSelection.fromRootStore.allAvailSpaces(this.sapi)
+  )
+  public allAvailableSpaces$ = this.store$.pipe(
+    atlasSelection.fromRootStore.allAvailSpaces(this.sapi)
+  )
+
```
Comment (xgui3783) [inline src/viewerModule/viewerCmp/viewerCmp.component.ts:115]: this cannot be right. 

the method says `allAvailSpaces`, it shouldn't return anything related to atlas

File: src/viewerModule/viewerCmp/viewerCmp.template.html [hunk 4]
```diff
@@ -376,12 +376,12 @@
...
-    quick-tour
-    [quick-tour-description]="quickTourAtlasSelector.description"
-    [quick-tour-order]="quickTourAtlasSelector.order">
-  </sxplr-sapiviews-core-atlas-dropdown-selector>
+<!--  <sxplr-sapiviews-core-atlas-dropdown-selector-->
+<!--    class="v-align-top sxplr-pt-2 pe-all mt-2 sxplr-custom-cmp bg card m-2 mat-elevation-z2 d-inline-block"-->
+<!--    quick-tour-->
+<!--    [quick-tour-description]="quickTourAtlasSelector.description"-->
+<!--    [quick-tour-order]="quickTourAtlasSelector.order">-->
+<!--  </sxplr-sapiviews-core-atlas-dropdown-selector>-->
```
Comment (xgui3783) [inline src/viewerModule/viewerCmp/viewerCmp.template.html:384]: remove properly

File: src/atlasComponents/sapiViews/core/atlas/chip/atlas.chip.component.ts (hunk 1 above)
Comment (fsdavid) [reply src/atlasComponents/sapiViews/core/atlas/chip/atlas.chip.component.ts]: I've removed chips for several reasons and fully removed these components

File: src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts (hunk 2 above)
Comment (fsdavid) [reply src/atlasComponents/sapiViews/core/atlas/smartChip/atlas.smartChip.components.ts:1]: Currently, all 3 components for atlas space parcellation are different enough to keep them separate

**Comment by fsdavid** [comment]
Moved here: https://github.com/FZJ-INM1-BDA/siibra-explorer/pull/1238

**Pull request closed without merging**